BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/MainActivity.kt

```diff
@@ -119,12 +122,12 @@ fun AlignYourBodyElement(
                 .size(88.dp)
                 .clip(CircleShape)
         )
+        Spacer(Modifier.height(8.dp))
         Text(
             text = stringResource(text),
-            style = MaterialTheme.typography.h3,
-            modifier = Modifier.paddingFromBaseline(
-                top = 24.dp, bottom = 8.dp
-            )
+            modifier = Modifier.paddingFromBaseline(top = 8.dp),
+            style = MaterialTheme.typography.bodyMedium,
+            color = MaterialTheme.colorScheme.onSurface
```

Comment:
Is there a reason we're customizing the color here where we weren't before?

Reply:
done